system-tests/projects/npm-react-webpack/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "devDependencies": {
+    "@babel/core": "7.4.5",
```

Comment:
Looks like we can remove a decent amount of these deps, namely:
```
"@bahmutov/cy-api": "1.4.2",
"@bahmutov/cy-rollup": "2.0.0",
"@cypress/code-coverage": "3.9.4",
"babel-plugin-istanbul": "6.0.0",
"babel-plugin-module-resolver": "4.0.0",
"cypress-plugin-snapshots": "1.4.4",
"module-resolver": "1.0.0",
```

Reply:
Done, nice.